docs/api_information/endpoint_locations.md

```diff
@@ -4,7 +4,7 @@ Environment<br/>Classification | Communication<br/>Protocol | Version | URL
 ----------|----------|----------|----------
 Dummy | SOAP | v1 | <a href="https://pos.oxipay.com.au/soap/v1/TestService.svc?wsdl">https://pos.oxipay.com.au/soap/v1/TestService.svc?wsdl</a>
 Dummy | HTTP | v1 | TBA
-TEST | SOAP | v1 | TBA
+TEST | SOAP | v1 | <a href="https://pos.oxipay.com.au/soap/v1/Service.svc?wsdl">https://pos.oxipay.com.au/soap/v1/Service.svc?wsdl</a>
```

Comment:
Just a note that I will have to get this changed ... This URL is more of the Prod URL .. the test should be testpos.oxipay.com.au ...